app/.gitignore

```diff
@@ -1,9 +1,7 @@
 # OSX
-#
```

Comment:
I would probably recommend against editing this file; when you go to upgrade React Native, the `react-native upgrade` command shows you a diff between the local and upgraded copies of the files, so the fewer files you edit the feweer changes you have to look through.